### PR TITLE
Java version update for 8, 11 and 17

### PR DESCRIPTION
**Pull request by yoshioterada**
I created this PR for following issue.
https://github.com/Azure/azure-functions-docker/issues/899

And I upload the Java version for Java 8, 11 and 17.

Releasing new Host versions.
[V4 Release](PASTE_V4_LINK_HERE)
[V3 Release](PASTE_V3_LINK_HERE)

---

### PR information

- [ ] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/development-docs/cleaning-up-commits.md).
- [x] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

**Comment by yoshioterada** [comment]
I don't know why v4 java build failed.
However in my local env, I executed following command and succeed to build all images except for `java*-composite.template`.

So I assume that it may be not related to this PR.

```bash
> pwd
$HOME/azure-functions-docker/host
```

```bash
> docker build -t 8build -f ./4/bullseye/amd64/java/java8/java8-build.Dockerfile .
docker build -t 8core -f ./4/bullseye/amd64/java/java8/java8-core-tools.Dockerfile .
docker build -t 8slim -f ./4/bullseye/amd64/java/java8/java8-slim.Dockerfile .
docker build -t 8prod -f ./4/bullseye/amd64/java/java8/java8.Dockerfile .

docker build -t 11build -f ./4/bullseye/amd64/java/java11/java11-build.Dockerfile .
docker build -t 11core -f ./4/bullseye/amd64/java/java11/java11-core-tools.Dockerfile .
docker build -t 11slim -f ./4/bullseye/amd64/java/java11/java11-slim.Dockerfile .
docker build -t 11prod -f ./4/bullseye/amd64/java/java11/java11.Dockerfile .

docker build -t 17build -f ./4/bullseye/amd64/java/java17/java17-build.Dockerfile .
docker build -t 17core -f ./4/bullseye/amd64/java/java17/java17-core-tools.Dockerfile .
docker build -t 17slim -f ./4/bullseye/amd64/java/java17/java17-slim.Dockerfile .
docker build -t 17prod -f ./4/bullseye/amd64/java/java17/java17.Dockerfile .
```

Following is the output

```output
[+] Building 0.2s (12/12) FINISHED                                                                                                                
 => [internal] load build definition from java8-build.Dockerfile                                                                             0.0s
 => => transferring dockerfile: 1.66kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0                                                                            0.1s
 => [1/7] FROM mcr.microsoft.com/dotnet/sdk:6.0@sha256:525bad260e88b16bc32adf5ea4e392e883a7a038163758e0a70664083f7bb37e                      0.0s
 => [internal] load build context                                                                                                            0.0s
 => => transferring context: 78B                                                                                                             0.0s
 => CACHED [2/7] RUN apt-get -qq update     && apt-get -qqy install curl     && apt-get install -y libfreetype6 fontconfig fonts-dejavu      0.0s
 => CACHED [3/7] RUN mkdir -p /usr/share/maven /usr/share/maven/ref   && curl -fsSL -o /tmp/apache-maven.tar.gz https://archive.apache.org/  0.0s
 => CACHED [4/7] RUN mkdir -p /usr/lib/jvm/adoptium-8-x64     && curl -fsSL -o /tmp/jdk.tar.gz https://github.com/adoptium/temurin8-binarie  0.0s
 => CACHED [5/7] COPY mvn-entrypoint.sh /usr/local/bin/mvn-entrypoint.sh                                                                     0.0s
 => CACHED [6/7] COPY settings-docker.xml /usr/share/maven/ref/                                                                              0.0s
 => CACHED [7/7] RUN chmod +x /usr/local/bin/mvn-entrypoint.sh                                                                               0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:ad0ab595a7b182e6381a7e4e7d674145841e3b1041d20cd11395f085c290cd01                                                 0.0s
 => => naming to docker.io/library/8build                                                                                                    0.0s
[+] Building 0.3s (6/6) FINISHED                                                                                                                  
 => [internal] load build definition from java8-core-tools.Dockerfile                                                                        0.0s
 => => transferring dockerfile: 2.69kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/azure-functions/java:4-java8-build                                                        0.2s
 => [1/2] FROM mcr.microsoft.com/azure-functions/java:4-java8-build@sha256:e1e5844d729b14147078a8ea5f6778cd6f62b45910137abe479a144a2faeacc5  0.0s
 => CACHED [2/2] RUN apt-get update     && apt-get -y install --no-install-recommends apt-utils dialog 2>&1     && apt-get -y install        0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:72e0968c3ffaac5f0a6e9b20740d6be10ebb84808cf7b1cae6e2f3a812510709                                                 0.0s
 => => naming to docker.io/library/8core                                                                                                     0.0s
[+] Building 0.2s (16/16) FINISHED                                                                                                                
 => [internal] load build definition from java8-slim.Dockerfile                                                                              0.0s
 => => transferring dockerfile: 4.51kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/dotnet/runtime-deps:6.0                                                                   0.1s
 => [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0                                                                            0.1s
 => [stage-1 1/7] FROM mcr.microsoft.com/dotnet/runtime-deps:6.0@sha256:73f9d4433218c21fd3a60661482424e91507cfae7248cb229e8714c627a10416     0.0s
 => [runtime-image 1/4] FROM mcr.microsoft.com/dotnet/sdk:6.0@sha256:525bad260e88b16bc32adf5ea4e392e883a7a038163758e0a70664083f7bb37e        0.0s
 => CACHED [stage-1 2/7] RUN apt-get update &&     apt-get install -y libc-dev                                                               0.0s
 => CACHED [stage-1 3/7] RUN apt-get update &&     apt-get install -y libfreetype6 fontconfig fonts-dejavu                                   0.0s
 => CACHED [runtime-image 2/4] RUN BUILD_NUMBER=$(echo 4.21.1 | cut -d'.' -f 3) &&     git clone --branch v4.21.1 https://github.com/Azure/  0.0s
 => CACHED [runtime-image 3/4] RUN apt-get update &&     apt-get install -y gnupg wget unzip &&     EXTENSION_BUNDLE_VERSION_V2=2.24.0 &&    0.0s
 => CACHED [runtime-image 4/4] RUN wget https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u372-b07/OpenJDK8U-jdk_x64_linu  0.0s
 => CACHED [stage-1 4/7] COPY --from=runtime-image [ /azure-functions-host, /azure-functions-host ]                                          0.0s
 => CACHED [stage-1 5/7] COPY --from=runtime-image [ /workers/java, /azure-functions-host/workers/java ]                                     0.0s
 => CACHED [stage-1 6/7] COPY --from=runtime-image [ /usr/lib/jvm/adoptium-8-x64, /usr/lib/jvm/adoptium-8-x64 ]                              0.0s
 => CACHED [stage-1 7/7] COPY --from=runtime-image [ /FuncExtensionBundles, /FuncExtensionBundles ]                                          0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:9735a323160520e43822275e204f6a553fb8d4f3021181fd9cd9b233e3f22699                                                 0.0s
 => => naming to docker.io/library/8slim                                                                                                     0.0s
[+] Building 0.4s (16/16) FINISHED                                                                                                                
 => [internal] load build definition from java8.Dockerfile                                                                                   0.0s
 => => transferring dockerfile: 4.51kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/dotnet/runtime-deps:6.0                                                                   0.3s
 => [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0                                                                            0.3s
 => [stage-1 1/7] FROM mcr.microsoft.com/dotnet/runtime-deps:6.0@sha256:73f9d4433218c21fd3a60661482424e91507cfae7248cb229e8714c627a10416     0.0s
 => [runtime-image 1/4] FROM mcr.microsoft.com/dotnet/sdk:6.0@sha256:525bad260e88b16bc32adf5ea4e392e883a7a038163758e0a70664083f7bb37e        0.0s
 => CACHED [stage-1 2/7] RUN apt-get update &&     apt-get install -y libc-dev                                                               0.0s
 => CACHED [stage-1 3/7] RUN apt-get update &&     apt-get install -y libfreetype6 fontconfig fonts-dejavu                                   0.0s
 => CACHED [runtime-image 2/4] RUN BUILD_NUMBER=$(echo 4.21.1 | cut -d'.' -f 3) &&     git clone --branch v4.21.1 https://github.com/Azure/  0.0s
 => CACHED [runtime-image 3/4] RUN apt-get update &&     apt-get install -y gnupg wget unzip &&     EXTENSION_BUNDLE_VERSION_V2=2.24.0 &&    0.0s
 => CACHED [runtime-image 4/4] RUN wget https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u372-b07/OpenJDK8U-jdk_x64_linu  0.0s
 => CACHED [stage-1 4/7] COPY --from=runtime-image [ /azure-functions-host, /azure-functions-host ]                                          0.0s
 => CACHED [stage-1 5/7] COPY --from=runtime-image [ /workers/java, /azure-functions-host/workers/java ]                                     0.0s
 => CACHED [stage-1 6/7] COPY --from=runtime-image [ /usr/lib/jvm/adoptium-8-x64, /usr/lib/jvm/adoptium-8-x64 ]                              0.0s
 => CACHED [stage-1 7/7] COPY --from=runtime-image [ /FuncExtensionBundles, /FuncExtensionBundles ]                                          0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:9735a323160520e43822275e204f6a553fb8d4f3021181fd9cd9b233e3f22699                                                 0.0s
 => => naming to docker.io/library/8prod                                                                                                     0.0s
[+] Building 0.2s (12/12) FINISHED                                                                                                                
 => [internal] load build definition from java11-build.Dockerfile                                                                            0.0s
 => => transferring dockerfile: 1.58kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0                                                                            0.1s
 => [internal] load build context                                                                                                            0.0s
 => => transferring context: 78B                                                                                                             0.0s
 => [1/7] FROM mcr.microsoft.com/dotnet/sdk:6.0@sha256:525bad260e88b16bc32adf5ea4e392e883a7a038163758e0a70664083f7bb37e                      0.0s
 => CACHED [2/7] RUN apt-get -qq update     && apt-get -qqy install curl     && apt-get install -y libfreetype6 fontconfig fonts-dejavu      0.0s
 => CACHED [3/7] RUN mkdir -p /usr/share/maven /usr/share/maven/ref   && curl -fsSL -o /tmp/apache-maven.tar.gz https://archive.apache.org/  0.0s
 => CACHED [4/7] RUN mkdir -p /usr/lib/jvm/msft-11-x64     && curl -fsSL -o /tmp/jdk.tar.gz https://aka.ms/download-jdk/microsoft-jdk-11.0.  0.0s
 => CACHED [5/7] COPY mvn-entrypoint.sh /usr/local/bin/mvn-entrypoint.sh                                                                     0.0s
 => CACHED [6/7] COPY settings-docker.xml /usr/share/maven/ref/                                                                              0.0s
 => CACHED [7/7] RUN chmod +x /usr/local/bin/mvn-entrypoint.sh                                                                               0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:fd9f727a5945f360ef497bb7525e6e48be9f3fcbf11fa8ee64fd7c5d6e095937                                                 0.0s
 => => naming to docker.io/library/11build                                                                                                   0.0s
[+] Building 0.3s (6/6) FINISHED                                                                                                                  
 => [internal] load build definition from java11-core-tools.Dockerfile                                                                       0.0s
 => => transferring dockerfile: 2.69kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/azure-functions/java:4-java11-build                                                       0.2s
 => [1/2] FROM mcr.microsoft.com/azure-functions/java:4-java11-build@sha256:b4e85dfe4211a529f7dd107c7f482e3e74d0d5c9cb6662f1cdb74f0aa4cbb0f  0.0s
 => CACHED [2/2] RUN apt-get update     && apt-get -y install --no-install-recommends apt-utils dialog 2>&1     && apt-get -y install        0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:50f810f4cf055dfa4611458dd77cd1395beeb1396acd39ffeb443015dc0fc4c7                                                 0.0s
 => => naming to docker.io/library/11core                                                                                                    0.0s
[+] Building 0.2s (16/16) FINISHED                                                                                                                
 => [internal] load build definition from java11-slim.Dockerfile                                                                             0.0s
 => => transferring dockerfile: 4.43kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/dotnet/runtime-deps:6.0                                                                   0.1s
 => [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0                                                                            0.1s
 => [stage-1 1/7] FROM mcr.microsoft.com/dotnet/runtime-deps:6.0@sha256:73f9d4433218c21fd3a60661482424e91507cfae7248cb229e8714c627a10416     0.0s
 => [runtime-image 1/4] FROM mcr.microsoft.com/dotnet/sdk:6.0@sha256:525bad260e88b16bc32adf5ea4e392e883a7a038163758e0a70664083f7bb37e        0.0s
 => CACHED [stage-1 2/7] RUN apt-get update &&     apt-get install -y libc-dev                                                               0.0s
 => CACHED [stage-1 3/7] RUN apt-get update &&     apt-get install -y libfreetype6 fontconfig fonts-dejavu                                   0.0s
 => CACHED [runtime-image 2/4] RUN BUILD_NUMBER=$(echo 4.21.1 | cut -d'.' -f 3) &&     git clone --branch v4.21.1 https://github.com/Azure/  0.0s
 => CACHED [runtime-image 3/4] RUN apt-get update &&     apt-get install -y gnupg wget unzip &&     EXTENSION_BUNDLE_VERSION_V2=2.24.0 &&    0.0s
 => CACHED [runtime-image 4/4] RUN wget https://aka.ms/download-jdk/microsoft-jdk-11.0.19-linux-x64.tar.gz &&     mkdir -p /usr/lib/jvm/msf  0.0s
 => CACHED [stage-1 4/7] COPY --from=runtime-image [ /azure-functions-host, /azure-functions-host ]                                          0.0s
 => CACHED [stage-1 5/7] COPY --from=runtime-image [ /workers/java, /azure-functions-host/workers/java ]                                     0.0s
 => CACHED [stage-1 6/7] COPY --from=runtime-image [ /usr/lib/jvm/msft-11-x64, /usr/lib/jvm/msft-11-x64 ]                                    0.0s
 => CACHED [stage-1 7/7] COPY --from=runtime-image [ /FuncExtensionBundles, /FuncExtensionBundles ]                                          0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:c07798a38a8a029890068672ef5ddaecb5f7becb6b6d1d9e4b2920cb24d25f7e                                                 0.0s
 => => naming to docker.io/library/11slim                                                                                                    0.0s
[+] Building 0.2s (16/16) FINISHED                                                                                                                
 => [internal] load build definition from java11.Dockerfile                                                                                  0.0s
 => => transferring dockerfile: 4.43kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/dotnet/runtime-deps:6.0                                                                   0.1s
 => [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0                                                                            0.1s
 => [stage-1 1/7] FROM mcr.microsoft.com/dotnet/runtime-deps:6.0@sha256:73f9d4433218c21fd3a60661482424e91507cfae7248cb229e8714c627a10416     0.0s
 => [runtime-image 1/4] FROM mcr.microsoft.com/dotnet/sdk:6.0@sha256:525bad260e88b16bc32adf5ea4e392e883a7a038163758e0a70664083f7bb37e        0.0s
 => CACHED [stage-1 2/7] RUN apt-get update &&     apt-get install -y libc-dev                                                               0.0s
 => CACHED [stage-1 3/7] RUN apt-get update &&     apt-get install -y libfreetype6 fontconfig fonts-dejavu                                   0.0s
 => CACHED [runtime-image 2/4] RUN BUILD_NUMBER=$(echo 4.21.1 | cut -d'.' -f 3) &&     git clone --branch v4.21.1 https://github.com/Azure/  0.0s
 => CACHED [runtime-image 3/4] RUN apt-get update &&     apt-get install -y gnupg wget unzip &&     EXTENSION_BUNDLE_VERSION_V2=2.24.0 &&    0.0s
 => CACHED [runtime-image 4/4] RUN wget https://aka.ms/download-jdk/microsoft-jdk-11.0.19-linux-x64.tar.gz &&     mkdir -p /usr/lib/jvm/msf  0.0s
 => CACHED [stage-1 4/7] COPY --from=runtime-image [ /azure-functions-host, /azure-functions-host ]                                          0.0s
 => CACHED [stage-1 5/7] COPY --from=runtime-image [ /workers/java, /azure-functions-host/workers/java ]                                     0.0s
 => CACHED [stage-1 6/7] COPY --from=runtime-image [ /usr/lib/jvm/msft-11-x64, /usr/lib/jvm/msft-11-x64 ]                                    0.0s
 => CACHED [stage-1 7/7] COPY --from=runtime-image [ /FuncExtensionBundles, /FuncExtensionBundles ]                                          0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:c07798a38a8a029890068672ef5ddaecb5f7becb6b6d1d9e4b2920cb24d25f7e                                                 0.0s
 => => naming to docker.io/library/11prod                                                                                                    0.0s
[+] Building 0.3s (12/12) FINISHED                                                                                                                
 => [internal] load build definition from java17-build.Dockerfile                                                                            0.0s
 => => transferring dockerfile: 1.58kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0                                                                            0.2s
 => [internal] load build context                                                                                                            0.0s
 => => transferring context: 78B                                                                                                             0.0s
 => [1/7] FROM mcr.microsoft.com/dotnet/sdk:6.0@sha256:525bad260e88b16bc32adf5ea4e392e883a7a038163758e0a70664083f7bb37e                      0.0s
 => CACHED [2/7] RUN apt-get -qq update     && apt-get -qqy install curl     && apt-get install -y libfreetype6 fontconfig fonts-dejavu      0.0s
 => CACHED [3/7] RUN mkdir -p /usr/share/maven /usr/share/maven/ref   && curl -fsSL -o /tmp/apache-maven.tar.gz https://archive.apache.org/  0.0s
 => CACHED [4/7] RUN mkdir -p /usr/lib/jvm/msft-17-x64     && curl -fsSL -o /tmp/jdk.tar.gz https://aka.ms/download-jdk/microsoft-jdk-17.0.  0.0s
 => CACHED [5/7] COPY mvn-entrypoint.sh /usr/local/bin/mvn-entrypoint.sh                                                                     0.0s
 => CACHED [6/7] COPY settings-docker.xml /usr/share/maven/ref/                                                                              0.0s
 => CACHED [7/7] RUN chmod +x /usr/local/bin/mvn-entrypoint.sh                                                                               0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:677a0edfa6f0109f0f837bf4a597ea1f59c5ebaf8d8e29324fc2e7a1d9aa5d34                                                 0.0s
 => => naming to docker.io/library/17build                                                                                                   0.0s
[+] Building 0.2s (6/6) FINISHED                                                                                                                  
 => [internal] load build definition from java17-core-tools.Dockerfile                                                                       0.0s
 => => transferring dockerfile: 2.69kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/azure-functions/java:4-java17-build                                                       0.1s
 => [1/2] FROM mcr.microsoft.com/azure-functions/java:4-java17-build@sha256:06c1a0c35c26809bd592f8edd32eded3e00c0152f165eddabd6166e606606cc  0.0s
 => CACHED [2/2] RUN apt-get update     && apt-get -y install --no-install-recommends apt-utils dialog 2>&1     && apt-get -y install        0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:50f58ac06d73463cf2dd0bac83e7c0dc162b4aa6e04e80e8bf671ada002becd4                                                 0.0s
 => => naming to docker.io/library/17core                                                                                                    0.0s
[+] Building 0.2s (16/16) FINISHED                                                                                                                
 => [internal] load build definition from java17-slim.Dockerfile                                                                             0.0s
 => => transferring dockerfile: 4.43kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/dotnet/runtime-deps:6.0                                                                   0.1s
 => [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0                                                                            0.1s
 => [stage-1 1/7] FROM mcr.microsoft.com/dotnet/runtime-deps:6.0@sha256:73f9d4433218c21fd3a60661482424e91507cfae7248cb229e8714c627a10416     0.0s
 => [runtime-image 1/4] FROM mcr.microsoft.com/dotnet/sdk:6.0@sha256:525bad260e88b16bc32adf5ea4e392e883a7a038163758e0a70664083f7bb37e        0.0s
 => CACHED [stage-1 2/7] RUN apt-get update &&     apt-get install -y libc-dev                                                               0.0s
 => CACHED [stage-1 3/7] RUN apt-get update &&     apt-get install -y libfreetype6 fontconfig fonts-dejavu                                   0.0s
 => CACHED [runtime-image 2/4] RUN BUILD_NUMBER=$(echo 4.21.1 | cut -d'.' -f 3) &&     git clone --branch v4.21.1 https://github.com/Azure/  0.0s
 => CACHED [runtime-image 3/4] RUN apt-get update &&     apt-get install -y gnupg wget unzip &&     EXTENSION_BUNDLE_VERSION_V2=2.24.0 &&    0.0s
 => CACHED [runtime-image 4/4] RUN wget https://aka.ms/download-jdk/microsoft-jdk-17.0.7-linux-x64.tar.gz &&     mkdir -p /usr/lib/jvm/msft  0.0s
 => CACHED [stage-1 4/7] COPY --from=runtime-image [ /azure-functions-host, /azure-functions-host ]                                          0.0s
 => CACHED [stage-1 5/7] COPY --from=runtime-image [ /workers/java, /azure-functions-host/workers/java ]                                     0.0s
 => CACHED [stage-1 6/7] COPY --from=runtime-image [ /usr/lib/jvm/msft-17-x64, /usr/lib/jvm/msft-17-x64 ]                                    0.0s
 => CACHED [stage-1 7/7] COPY --from=runtime-image [ /FuncExtensionBundles, /FuncExtensionBundles ]                                          0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:6d51bfcb5a8a3028cfb2b17d8d45c7218d481f04bd342bf178249deb91f0c731                                                 0.0s
 => => naming to docker.io/library/17slim                                                                                                    0.0s
[+] Building 0.2s (16/16) FINISHED                                                                                                                
 => [internal] load build definition from java17.Dockerfile                                                                                  0.0s
 => => transferring dockerfile: 4.43kB                                                                                                       0.0s
 => [internal] load .dockerignore                                                                                                            0.0s
 => => transferring context: 2B                                                                                                              0.0s
 => [internal] load metadata for mcr.microsoft.com/dotnet/runtime-deps:6.0                                                                   0.1s
 => [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0                                                                            0.1s
 => [stage-1 1/7] FROM mcr.microsoft.com/dotnet/runtime-deps:6.0@sha256:73f9d4433218c21fd3a60661482424e91507cfae7248cb229e8714c627a10416     0.0s
 => [runtime-image 1/4] FROM mcr.microsoft.com/dotnet/sdk:6.0@sha256:525bad260e88b16bc32adf5ea4e392e883a7a038163758e0a70664083f7bb37e        0.0s
 => CACHED [stage-1 2/7] RUN apt-get update &&     apt-get install -y libc-dev                                                               0.0s
 => CACHED [stage-1 3/7] RUN apt-get update &&     apt-get install -y libfreetype6 fontconfig fonts-dejavu                                   0.0s
 => CACHED [runtime-image 2/4] RUN BUILD_NUMBER=$(echo 4.21.1 | cut -d'.' -f 3) &&     git clone --branch v4.21.1 https://github.com/Azure/  0.0s
 => CACHED [runtime-image 3/4] RUN apt-get update &&     apt-get install -y gnupg wget unzip &&     EXTENSION_BUNDLE_VERSION_V2=2.24.0 &&    0.0s
 => CACHED [runtime-image 4/4] RUN wget https://aka.ms/download-jdk/microsoft-jdk-17.0.7-linux-x64.tar.gz &&     mkdir -p /usr/lib/jvm/msft  0.0s
 => CACHED [stage-1 4/7] COPY --from=runtime-image [ /azure-functions-host, /azure-functions-host ]                                          0.0s
 => CACHED [stage-1 5/7] COPY --from=runtime-image [ /workers/java, /azure-functions-host/workers/java ]                                     0.0s
 => CACHED [stage-1 6/7] COPY --from=runtime-image [ /usr/lib/jvm/msft-17-x64, /usr/lib/jvm/msft-17-x64 ]                                    0.0s
 => CACHED [stage-1 7/7] COPY --from=runtime-image [ /FuncExtensionBundles, /FuncExtensionBundles ]                                          0.0s
 => exporting to image                                                                                                                       0.0s
 => => exporting layers                                                                                                                      0.0s
 => => writing image sha256:6d51bfcb5a8a3028cfb2b17d8d45c7218d481f04bd342bf178249deb91f0c731                                                 0.0s
 => => naming to docker.io/library/17prod                                                                                                    0.0s
```

```bash
> docker images
REPOSITORY                                                         TAG                    IMAGE ID       CREATED          SIZE
17prod                                                             latest                 6d51bfcb5a8a   6 minutes ago    1.57GB
17slim                                                             latest                 6d51bfcb5a8a   6 minutes ago    1.57GB
afunc8                                                             latest                 6d51bfcb5a8a   6 minutes ago    1.57GB
17core                                                             latest                 50f58ac06d73   10 minutes ago   2.7GB
17build                                                            latest                 677a0edfa6f0   13 minutes ago   1.1GB
11prod                                                             latest                 c07798a38a8a   15 minutes ago   1.56GB
11slim                                                             latest                 c07798a38a8a   15 minutes ago   1.56GB
11core                                                             latest                 50f810f4cf05   22 minutes ago   2.69GB
11build                                                            latest                 fd9f727a5945   25 minutes ago   1.09GB
8prod                                                              latest                 9735a3231605   27 minutes ago   1.44GB
8slim                                                              latest                 9735a3231605   27 minutes ago   1.44GB
8core                                                              latest                 72e0968c3ffa   32 minutes ago   2.57GB
8build                                                             latest                 ad0ab595a7b1   37 minutes ago   969MB
```

And I confirmed the update version is runnings.

For Java 8

```bash
> docker exec 1d26218f140b  /usr/lib/jvm/adoptium-8-x64/bin/java -version 
openjdk version "1.8.0_372"
OpenJDK Runtime Environment (Temurin)(build 1.8.0_372-b07)
OpenJDK 64-Bit Server VM (Temurin)(build 25.372-b07, mixed mode)
> 
```

For Java 11

```bash
> docker exec 1a2266daf2c1 /usr/lib/jvm/msft-11-x64/bin/java -version
openjdk version "11.0.19" 2023-04-18 LTS
OpenJDK Runtime Environment Microsoft-7621296 (build 11.0.19+7-LTS)
OpenJDK 64-Bit Server VM Microsoft-7621296 (build 11.0.19+7-LTS, mixed mode)
```

For Java 17 

```bash
> docker exec 2f90863233fd /usr/lib/jvm/msft-17-x64/bin/java -version
openjdk version "17.0.7" 2023-04-18 LTS
OpenJDK Runtime Environment Microsoft-7626293 (build 17.0.7+7-LTS)
OpenJDK 64-Bit Server VM Microsoft-7626293 (build 17.0.7+7-LTS, mixed mode, sharing)
```

File: host/4/bullseye/amd64/java/java8/java8-composite.template [hunk 1]
```diff
@@ -1,5 +1,5 @@
-ARG JAVA_VERSION=8u322b06
-ARG JDK_NAME=jdk8u322-b06
+ARG JAVA_VERSION=8u372b07
```
Comment (kaibocai) [inline host/4/bullseye/amd64/java/java8/java8-composite.template]: This one is missed

File: host/3.0/buster/amd64/java/java8/java8-build.Dockerfile [hunk 2]
```diff
@@ -4,8 +4,8 @@ ARG MAVEN_VERSION=3.8.5
 ARG USER_HOME_DIR="/root"
 ARG SHA=89ab8ece99292476447ef6a6800d9842bbb60787b9b8a45c103aa61d2f205a971d8c3ddfb8b03e514455b4173602bd015e82958c0b3ddc1728a57126f773c743
 ARG BASE_MAVEN_URL=https://archive.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries
-ARG JAVA_VERSION=8u322b06
-ARG JDK_NAME=jdk8u322-b06
+ARG JAVA_VERSION=8u362-b09
```
Comment (kaibocai) [inline host/3.0/buster/amd64/java/java8/java8-build.Dockerfile]: JAVA_VERSION should be 8u362b09

File: host/4/bullseye/amd64/java/java8/java8-composite.template (hunk 1 above)
Comment (yoshioterada) [reply host/4/bullseye/amd64/java/java8/java8-composite.template]: Thank you so much !! That's right!!
And I'm sorry.
I mistook to read. I saw the title but file name.

https://github.com/adoptium/temurin8-binaries/releases/tag/jdk8u362-b09

File: host/3.0/buster/amd64/java/java8/java8-build.Dockerfile (hunk 2 above)
Comment (yoshioterada) [reply host/3.0/buster/amd64/java/java8/java8-build.Dockerfile]: That's right.  I will modifiy them. Thank you so much !!

**Comment by yoshioterada** [comment]
Dear @kaibocai san, I updated the version, so could you confirm them again please?

**Comment by kaibocai** [comment]
> Dear @kaibocai san, I updated the version, so could you confirm them again please?

Yeah I will work on it. Thanks.

**Comment by kaibocai** [comment]
New PR opened at https://github.com/Azure/azure-functions-docker/pull/902. Close this one.

**Pull request closed without merging**